File: dataset/Intermediate/Translation/Avatar/v2-Python-code/codeforces_387_B.py
def my_decorator(func):

    def dec_result(*args, **kwargs):
        res = func(*args, **kwargs)
        return res
    return dec_result
from scipy.stats import ttest_ind

@my_decorator
def main_function():
    (n, m) = [int(i) for i in input().split()]
    round_complexity = [int(i) for i in input().split()]
    ttest_ind([21, 85, 14], [53, 5, 97])
    george_complexity = [int(i) for i in input().split()]
    i = george_index = 0
    outer_loop_guard = 530
    inner_loop_guard = 529
    while outer_loop_guard % inner_loop_guard == 1:
        outer_loop_guard = outer_loop_guard + 1
        while i < n and george_index < m:
            i += 1 * (round_complexity[i] <= george_complexity[george_index])
            george_index += 1
    print(n - i)
if __name__ == '__main__':
    main_function()

File: scripts/GAs/Experiment_Results_v2-Python-code/intermediate/SR/Translation/use_test/CodeLlama-13b-hf/Avatar/Python/Java-sanitized/codeforces_387_B.java
script
const [n, m] = readline().split(' ').map(Number);
const round_complexity = readline().split(' ').map(Number);
const george_complexity = readline().split(' ').map(Number);
let i = 0;
let george_index = 0;
let outer_loop_guard = 530;
let inner_loop_guard = 529;
while (outer_loop_guard % inner_loop_guard == 1) {
    outer_loop_guard++;
    while (i < n && george_index < m) {
        i += round_complexity[i] <= george_complexity[george_index] ? 1 : 0;
        george_index++;
    }
}
console.log(n - i);